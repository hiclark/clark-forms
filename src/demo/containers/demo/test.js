import React from 'react';
import configureStore from 'redux-mock-store';

import Demo from './';

const initialState = {
  form: {},
};
const mockStore = configureStore();
let wrapper;
let store;

beforeEach(() => {
  store = mockStore(initialState);
  wrapper = shallow(<Demo store={store} />);
});

it('snapshot test <Demo />', () => {
  expect(wrapper).toMatchSnapshot();
});
