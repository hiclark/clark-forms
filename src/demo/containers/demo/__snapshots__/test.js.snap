// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test <Demo /> 1`] = `
<Connect(Form(Demo))
  destroyOnUnmount={false}
  enableReinitialize={false}
  forceUnregisterOnUnmount={true}
  form="demo"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
/>
`;
